contentcuration/contentcuration/frontend/shared/views/errors/ChannelDeletedError.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+
+  <AppError>
+    <template #header>
+      {{ $tr('channelDeletedHeader') }}
+    </template>
+    <template #details>
+      {{ $tr('channelDeletedDetails') }}
+    </template>
+    <template #actions>
+      <VBtn v-bind="backHomeLink">
+        {{ $tr('backToHomeAction') }}
+      </VBtn>
+      <VBtn v-bind="restoreChannelLink" color="primary">
```

Comment:
Could you remove this button? The admin page is for LE only, so surfacing this button could open potential security risks for us with people knowing about the admin page in general